deflatebench.py

```
@@ -227,9 +223,28 @@ def benchmain():
-        results[level] = []
+        result_comp[level] = []
+        result_decomp[level] = []
+
+    # Prepare compressed files for 'benchmode decomp only' mode
+    if benchmode == 'decomp' and cfgRuns['testmode'] != 'multi':
+        printnn("Compressing tempfiles for decompression test ")
+        if cfgRuns['testmode'] == 'single':
+            srcfile = cfgSingle['testfile']
+        else: # gen
+            srcfile = cfgGen['srcFile']
+        compfile = util.findfile(srcfile)
```

Ruff (0.14.14)

[error] Local variable `compfile` is assigned to but never used

Remove assignment to unused variable `compfile`

(F841)